### PR TITLE
Fix verification

### PR DESCRIPTION
**Pull request by AlexD10S**

### CODE REVIEW

**Comment by maurolucc** [comment]
Update dependencies (minor/patch). It will fix jose vulnerability (see dependabot alert).

```
npx npm-check -u
```

File: src/util/Util.ts [hunk 1]
```diff
@@ -142,6 +142,13 @@ const DidMatchFromJwksUri = (
   return jwksUri.includes(issuerDid);
 };
 
+const compareKidWithId = (kid: string, elem: VerificationMethod): boolean => {
+  if (kid.includes("did:") || kid.startsWith("#")) {
+    return elem.id === kid;
+  }
```
Comment (xvilapueyo) [inline src/util/Util.ts:148]: Heads up: if `kid = #123`and `elem = [{id: did:key:123#123}]` this would fail, right?

File: package.json [hunk 2]
```diff
@@ -78,6 +78,7 @@
   },
   "dependencies": {
     "@transmute/did-key-ed25519": "^0.2.1-unstable.31",
+    "@transmute/did-key.js": "*",
```
Comment (maurolucc) [inline package.json:81]: 🤔

Comment (AlexD10S) [reply package.json]: update to "^0.2.1-unstable.31"

**Comment by maurolucc** [comment]
All tests passed ✅

**Review by maurolucc — APPROVED** [review]

**Pull request merged**